tests/backends/matplotlib/mpl_image_test.py

```diff
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+
+from plopp.data.testing import data_array
+from plopp.backends.matplotlib.canvas import Canvas
+from plopp.backends.matplotlib.image import Image
+import scipp as sc
+
+
+def test_image_creation():
+    da = data_array(ndim=2)
+    image = Image(canvas=Canvas(), data=da)
+    assert sc.identical(image._data, da)
+
+
+def test_image_creation_binedges():
+    da = data_array(ndim=2, binedges=True)
+    image = Image(canvas=Canvas(), data=da)
+    assert sc.identical(image._data, da)
+
+
+def test_image_creation_masks():
+    da = data_array(ndim=2, masks=True)
+    image = Image(canvas=Canvas(), data=da)
+    assert sc.identical(image._data, da)
+
+
+def test_image_creation_ragged_coord():
+    da = data_array(ndim=2, ragged=True)
+    image = Image(canvas=Canvas(), data=da)
+    assert image._dim_2d == ('x', 'xx')
+    assert image._dim_1d == ('y', 'yy')
```

Comment:
Not sure about the value of any of these tests. We are testing some private properties, or deeply nested attributes. Future refactoring may turn out to be painful.

That being said, I understand that testing plots is challenging, and I don't have a better suggestion.